python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -40,6 +40,12 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     yield from dbt.cli(["build"], context=context).stream()
 
 
+def _get_select_args(dbt_cli_invocation) -> Set[str]:
+    args = list(dbt_cli_invocation.process.args)
+    assert args[-2] == "--select"
+    return set(args[-1].split())
```

Comment:
super nit: I'm allergic to indexing directly into a list. Let's just deconstruct?

```suggestion
    *_, dbt_select_flag, dbt_select_args = list(dbt_cli_invocation.process.args)
    
    assert dbt_select_flag == "--select"
    
    return set(dbt_select_args.split())
```